src/Workspaces/CoreTestUtilities/Microsoft.CodeAnalysis.Workspaces.Test.Utilities.csproj

```diff
@@ -8,6 +8,9 @@
     <UseWpf>true</UseWpf>
     <IsShipping>false</IsShipping>
     <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
+
+    <!-- CSC error AD0001: Analyzer 'Microsoft.CodeAnalysis.CSharp.BannedApiAnalyzers.CSharpRestrictedInternalsVisibleToAnalyzer' threw an exception of type 'System.NullReferenceException' with message 'Object reference not set to an instance of an object.'. -->
```

Comment:
Is there a bug tracking this?

Reply:
Yes, see https://github.com/dotnet/roslyn/pull/48883#discussion_r511207097

Reply:
We generally put these in source as well.

Reply:
I'm going to try and fix it as part of an in-progress change to the analyzer, and otherwise comment it.